2021-09-29 18:00:57,752 INFO    MainThread:11118 [wandb_setup.py:_flush():69] setting env: {}
2021-09-29 18:00:57,752 INFO    MainThread:11118 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-29 18:00:57,752 INFO    MainThread:11118 [wandb_init.py:_log_setup():348] Logging user logs to /opt/ml/detection/mmdetection/wandb/run-20210929_180057-2lhh28g9/logs/debug.log
2021-09-29 18:00:57,752 INFO    MainThread:11118 [wandb_init.py:_log_setup():349] Logging internal logs to /opt/ml/detection/mmdetection/wandb/run-20210929_180057-2lhh28g9/logs/debug-internal.log
2021-09-29 18:00:57,753 INFO    MainThread:11118 [wandb_init.py:_jupyter_setup():300] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f01c1a92c10>
2021-09-29 18:00:57,754 INFO    MainThread:11118 [wandb_init.py:init():381] calling init triggers
2021-09-29 18:00:57,754 INFO    MainThread:11118 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-09-29 18:00:57,754 INFO    MainThread:11118 [wandb_init.py:init():430] starting backend
2021-09-29 18:00:57,754 INFO    MainThread:11118 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-29 18:00:57,783 INFO    MainThread:11118 [backend.py:ensure_launched():135] starting backend process...
2021-09-29 18:00:57,814 INFO    MainThread:11118 [backend.py:ensure_launched():140] started backend process with pid: 11764
2021-09-29 18:00:57,817 INFO    MainThread:11118 [wandb_init.py:init():435] backend started and connected
2021-09-29 18:00:57,850 INFO    MainThread:11118 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-09-29 18:00:57,852 INFO    MainThread:11118 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-29 18:00:57,852 INFO    MainThread:11118 [wandb_init.py:init():494] updated telemetry
2021-09-29 18:00:57,853 INFO    MainThread:11118 [wandb_init.py:init():517] communicating current version
2021-09-29 18:00:58,554 INFO    MainThread:11118 [wandb_init.py:init():522] got version response 
2021-09-29 18:00:58,554 INFO    MainThread:11118 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-29 18:00:58,784 INFO    MainThread:11118 [wandb_init.py:init():557] starting run threads in backend
2021-09-29 18:01:03,791 INFO    MainThread:11118 [wandb_run.py:_console_start():1605] atexit reg
2021-09-29 18:01:03,792 INFO    MainThread:11118 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-09-29 18:01:03,792 INFO    MainThread:11118 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-09-29 18:01:03,793 INFO    MainThread:11118 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-29 18:01:03,793 INFO    MainThread:11118 [wandb_init.py:init():582] run started, returning control to user process
2021-09-29 18:06:05,036 INFO    MainThread:11118 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-29 18:06:05,037 INFO    MainThread:11118 [jupyter.py:save_ipynb():301] looking for notebook: mmdetection/mm_vfNet_train.ipynb
2021-09-29 18:06:09,758 INFO    MainThread:11118 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-09-29 18:06:09,759 INFO    MainThread:11118 [wandb_run.py:_restore():1547] restore
